src/core/resources/test-reader.cfg

```diff
@@ -0,0 +1,6 @@
+sensor:
```

Comment:
czytales reszte kodu? to jest przykladowy konfig do testów jednostkowych. to jest PR tylko do tego konkretnego modulu, poprawki wszystkich "dummy method, read from config here" beda robione na oddzielnym branchu (albo może od razu do mastera, bo to trywialne) i z oddzielnym PR.